aws/policy/storage-services.yaml

```diff
@@ -27,6 +27,7 @@ Statement:
       - s3:PutBucketRequestPayment
       - s3:PutBucketTagging
       - s3:PutBucketVersioning
+      - s3:PutBucketAccelerateConfiguration
```

Comment:
This needs to be `PutAccelerateConfiguration`, and should be ordered alphabetically in the policy.